helm-chart/renku-core/values.yaml

```diff
@@ -78,23 +78,23 @@ resources: {}
 
 # affinity: {}
 
-## The renku core service deploys its own redis as a dependency. These values
-## do probably not need to be changed.
-redis:
-  nameOverride: core-redis
+## The renku core service uses the global redis instance of the renku platform.
```

Comment:
@Panaetius this whole section should be removed fully. The "hack" with `call-nested` essentially allows us to deploy this with renku. But you still need to specify all the other things in `.global.redis` - regardless of whether you have redis as a subchart or not.

If you want to be able to deploy with the redis instance defined here then we need to change the chart further.

Reply:
I saw you kept this around in the gateway PR and thought we can do the same. but I don't think we want to deploy with an internal redis, so I'll remove it